src/components/card/card.component.scss

```diff
@@ -4,6 +4,7 @@
   display: flex;
   flex-direction: column;
   align-items: stretch;
+  width: 854px;
```

Comment:
magic number?

Reply:
I'll make a variable for the magic number.